core/src/blocks/database.rs

```diff
@@ -67,60 +68,42 @@ impl Block for Database {
         let config = env.config.config_for_block(name);
 
         let err_msg = format!(
-            "Invalid or missing `database` in configuration for \
-        `database` block `{}` expecting `{{ \"database\": \
-        {{ \"workspace_id\": ..., \"data_source_id\": ..., \"database_id\": ... }} }}`",
+            "Invalid or missing `tables` in configuration for \
```

Comment:
block's config now expect a `tables: {workspace_id, datasource_id, table_id}[]` instead of `database: {..}`.